include/util.h

```diff
@@ -62,8 +77,14 @@ netplan_util_dump_yaml_subtree(const char* prefix, int input_fd, int output_fd,
 
 /********** Old API below this ***********/
 
+/**
+ * \deprecated Make use of `netplan_netdef_get_filepath()` instead.
```

Comment:
```suggestion
 * \deprecated Use of `netplan_netdef_get_filepath()` instead.
```

Reply:
This should probably be just "Use `netplan..."